workflows/transport/pika_transport.py

```diff
@@ -105,7 +110,15 @@ def __call__(self, parser, namespace, value, option_string=None):
             type=str,
             action=SetParameter,
         )
-        # prefix
+
+        argparser.add_argument(
+            "--rabbit-prfx",
+            metavar="PRE",
+            default=cls.defaults.get("--rabbit-prfx"),
+            help="Rabbit namespace prefix, default '%(default)s'",
```

Comment:
Suggest renaming this to virtual hosts, and the parameter to `--rabbit-vhost` - we can use the words that a RabbitMQ user might understand.

Reply:
Thanks!